content/articles/premium-domains.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Premium Domains
+excerpt: In the new gTLDs domain space, a premium domain is a domain that has been considered of special value by a registrar.
+categories:
+- Domains
+---
+
+# Premium Domains
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## What is a Premium Domain?
+
+A premium domain is a domain that has been considered of special value by a registrar. Premium domains generally are available in the new gTLDs domain space. They are usually more expensive that regular domains, prices going up to several thousand dollars for one domain in some cases.
+
+## What makes a Premium Domain valuable
+
+Premium domains appeal to specific businesses or groups of people providing them very with useful, specific domain names for their activities. Other times they are just popular keywords or a very well known combination of name and domain. Registrars also consider every two or three leter new gTLDs as premium, there is value on being very short.
+
+All of this is better illustrated with an example: Paris is worldwide known as the city of love, the domain created as a combination of paris and love therefore the domain `paris.love` is a very attractive one for marketeers, business in the city or even the city itself and may be flagged by the registry as premium domain.
```

Comment:
Would suggest: "Here is an example: Paris is known as the city of love. The domain created as a combination of these could be `paris.love` and would be a very attractive name for marketing companies or businesses. The `.love` registry therefore may impose a higher fee for this rare and beautiful combination and the registrar may optionally add to this price to make it more expensive than a typical `.love` domain registration."